docs/BuildFromSource.md

```diff
@@ -81,21 +81,24 @@ Instead, we have many .sln files which include a sub-set of projects. These prin
 
 > :bulb: Pro tip: `dotnet new sln` and `dotnet sln` are one of the easiest ways to create and modify solutions.
 
-### Known issue: NU1105
+### Common error: CS0006
```

Comment:
Can external contributors who are not themselves adding a new project hit this? That is, doesn't `CodeCheck.ps1` prevent seeing this error in 'master'?

Reply:
Yes. CodeCheck.ps1 only checks that each .csproj referenced from a .sln exists. It doesn't check that the entire project reference graph has been added to the solution.